doc/design.md

```diff
@@ -142,6 +142,17 @@ Covers:
 
 Needs: impl, itest
 
+### ExaLoader in Common Docker Network
+`dsn~exaloader-in-common-docker-network~1`
+
+ETC allows running two test container in the same docker network so that the ExaLoader is able to execute `IMPORT` statements that import data from one database into the other.
```

Comment:
```suggestion
ETC allows running two test containers in the same docker network so that the ExaLoader is able to execute `IMPORT` statements that import data from one database into the other.
```

Reply:
Fixed.